packages/config/config/devices/0x0258/motion_sensor_2.json

```
@@ -0,0 +1,158 @@
+		},
+		"2": {
+			"label": "ON/OFF Duration Setting",
+			"description": "Determines how long the associated devices should stay ON status",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 30
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"description": "Basic Set Command will be sent when motion is triggered",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		"4": {
+			"label": "Enable/Disable PIR Function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable PIR Detector Function",
+					"value": 0
+				},
+				{
+					"label": "Enable PIR Detector Function",
+					"value": 255
+				}
+			]
+		},
+		"5": {
+			"label": "Lux Level Set",
+			"description": "Set a lux level value which determines when the light sensor is activated.",
+			"unit": "LUX",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 100
+		},
+		"6": {
+			"label": "Re-trigger Interval",
+			"description": "Adjust the interval of PIR being re-triggered",
+			"unit": "Seconds",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8
+		},
+		"7": {
+			"label": "Light Sensor Polling Interval",
+			"description": "The Interval after which the Light Sensor Measure is polled.",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 36000,
+			"defaultValue": 180,
+			"unsigned": true
+		},
+		"8": {
+			"label": "Lux Level Function Enable",
+			"description": "0 = Off, 1 =On",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable Lux Level Function",
+					"value": 0
+				},
+				{
+					"label": "Enable Lux Level Function",
+					"value": 1
+				}
+			]
+		},
+		"9": {
+			"label": "Lux Level Changed Report",
+			"description": "How much the Lux Level must change",
+			"unit": "LUX",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Temperature Differential Report",
+			"description": "Differential between current & previous report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 5
+		},
+		"11": {
+			"label": "LED Blink Enable",
+			"description": "Defines whether the LED blinking is on / off",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable LED blinking",
+					"value": 0
+				},
+				{
+					"label": "Enable LED blinking",
+					"value": 1
+				}
+			]
+		},
+		"99": {
+			"label": "Ambient light intensity calibration",
+			"description": "Define the calibrated scale for ambient light intensity",
```

```suggestion
			"label": "Ambient Light Intensity Calibration",
```
I don't understand the manual here 🤷🏻‍♂️